area/midgaard.are

```diff
@@ -174,19 +174,27 @@ AFN ABCD 0 0
 stand stand female 4500
 0 0 medium 0
 #3011
-frank bruno~
-Frank Bruno~
-Frank Bruno stands here protecting Xania.
+Concordius~
+Concordius~
+Concordius stands here keeping the peace.
 ~
-Frank Bruno is covered in muscle, he is ready to fight and die defending Xania
-If you are a killer or thief you should beware as he will destroy you.
+Concordius is a tall imposing figure with a weathered face that suggests great
+experience. 
 
-                     ...BEHAVE WELL OR FACE HIM...
+Word has it that Concordius is some kind of arbiter, or arbitrator, 
+or was it arboretum? You didn't quite catch what the guy said.
 
+A lightweight toga spans his lean body. An intricate claw-like
+object adorns to his right hand, and a bronze shield depicting a sun is 
+strapped to his left arm.
+
+Although he seems to be quite relaxed, his upright posture and alertness
+indicate that he is ready to respond at a moment's notice.
+As he speaks, his sonorous voice leaves you feeling enlightened.
 ~
-giant~
-BMTV CDFJZ 1000 0
-86 30 1d1+6999 1d1+999 4d10+30 punch
+human~
+BMTV CDFHJZ 1000 0
```

Comment:
Stats very similar to the old one, except +Sanctuary.

Reply:
Seems appropriate